pandas/tests/reductions/test_reductions.py

```diff
@@ -912,8 +912,13 @@ def test_all_any_boolean(self):
             index=[0, 0, 1, 1, 2, 2],
             dtype="boolean",
         )
-        tm.assert_series_equal(s.all(level=0), Series([False, True, False]))
-        tm.assert_series_equal(s.any(level=0), Series([False, True, True]))
+        result = s.all(level=0)
```

Comment:
can you pull L910 out to a new test. Then parameterize on bool and boolean.

Reply:
okay